__tests__/app.test.js

```diff
@@ -127,24 +131,40 @@ describe("Articles", () => {
           expect(result.body.msg).toBe("400 - Invalid Request");
         });
     });
-    test("200 - valid article id is provided and votes updates by 1 correctly", () => {
+    test("404 - article id is not in database", () => {
       return request(app)
-        .patch("/api/articles/1")
+        .patch("/api/articles/99999")
         .send({ inc_votes: 1 })
-        .expect(201)
+        .expect(404)
         .then((result) => {
-          expect(result.body.votes).toBe(101);
+          expect(result.body.msg).toBe("404 - Article Not Found");
         });
     });
-    test("200 - valid article id is provided and votes updates by 100 correctly", () => {
+  });
+
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("200 article_id is valid and comments exist", () => {
       return request(app)
-        .patch("/api/articles/1")
-        .send({ inc_votes: 100 })
-        .expect(201)
+        .get("/api/articles/5/comments")
+        .expect(200)
         .then((result) => {
-          expect(result.body.votes).toBe(200);
+          let { comments } = result.body;
+          expect(Array.isArray(comments)).toBe(true);
+          expect(comments).toHaveLength(2);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            });
+          });
         });
     });
+    test("200 article_id is valid and no comments exist", () => {
+      return request(app).get("/api/articles/2/comments").expect(200);
+    });
```

Comment:
need to add a couple of tests for errors here
think about incorrect article id, non existent and invalid